### DISCUSSION TITLE
Is it possible is compute contrast from glmsingle output?

### DISCUSSION THREAD

**Discussion by cnwangy** [cvnlab/GLMsingle#21] [Q&A] [answered]
I wonder if it's possible to compute contrast from glmsingle output. Is there any modification needed?

**Comment by kendrickkay** [comment, marked as answer]
GLMsingle provides estimates of the BOLD response amplitude for each trial in the experiment.  So, it's up to the user of the outputs of GLMsingle to compute contrasts (which I assume you mean something like doing a two-sample t-test that compares the magnitude of the response to different trial groups).  You'll have to write some code to do that...
